src/output_stream.rs

```diff
@@ -179,9 +186,30 @@ impl<O: IsA<OutputStream> + IsA<glib::Object> + Clone + 'static> OutputStreamExt
     }
 }
 
+pub struct OutputStreamWrite<T: OutputStreamExt>(T);
```

Comment:
Should implement `Eq` and `Debug` here at least, based on the impls of the contained type. Can probably be derived